docs/Developers/Newcomers_Guide.rst

```diff
@@ -62,7 +62,9 @@ Step 0. Run coala
 As a preparation of joining the community you should find out what this project
 is about - if you didn't do this already. We highly recommend you
 `install coala <https://coala.io/install>`_ and use it on at least one of your
-projects.
+projects. Also, we recommend that you read
+`Development Setup Notes <https://api.coala.io/en/latest/Developers/Development_Setup.html>`
+for setting up an environment to work on coala.
```

Comment:
"for setting up an environment" -> "to learn how to setup an environment"
(suggestion)